airbyte-integrations/connectors/destination-bigquery/src/main/java/io/airbyte/integrations/destination/bigquery/typing_deduping/BigQueryDestinationHandler.java

```diff
@@ -112,7 +114,7 @@ SELECT TIMESTAMP_SUB(MIN(_airbyte_extracted_at), INTERVAL 1 MICROSECOND)
             SELECT MAX(_airbyte_extracted_at)
             FROM ${raw_table}
             """))
-        .build()).iterateAll().iterator().next().get(0);
```

Comment:
this is ugly as sin. Are you sure we didn't forget to iterate on  the iterateAll iterator?